srcpkgs/python3-httplib2/template

```diff
@@ -1,26 +1,29 @@
 # Template file for 'python3-httplib2'
 pkgname=python3-httplib2
-version=0.18.1
-revision=5
-create_wrksrc=yes
-build_wrksrc="httplib2-${version}"
+version=0.21.0
+revision=1
 build_style=python3-module
 hostmakedepends="python3-setuptools"
-depends="python3 ca-certificates"
+depends="python3 ca-certificates python3-parsing"
+checkdepends="python3-pytest python3-pytest-cov python3-pytest-timeout
+ python3-cryptography python3-flake8 python3-six python3-future"
 short_desc="Comprehensive HTTP client library (Python3)"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="MIT"
 homepage="https://github.com/httplib2/httplib2"
 changelog="https://raw.githubusercontent.com/httplib2/httplib2/master/CHANGELOG"
-distfiles="${PYPI_SITE}/h/httplib2/httplib2-${version}.tar.gz
- https://raw.githubusercontent.com/httplib2/httplib2/master/LICENSE>LICENSE.txt"
-checksum="8af66c1c52c7ffe1aa5dc4bcd7c769885254b0756e6e69f953c7f0ab49a70ba3
- 589eec38f72df2be203711d3b8cbece9b908c5e7ff00bc3cab7f63bae9e366b4"
+distfiles="${PYPI_SITE}/h/httplib2/httplib2-${version}.tar.gz"
+checksum=fc144f091c7286b82bec71bdbd9b27323ba709cc612568d3000893bfd9cb4b34
+
+do_check() {
+	# https://github.com/httplib2/httplib2/issues/221
+	PYTHONPATH="$PWD/build/lib" python3 -m pytest tests/ -k 'not test_client_cert_password_verified'
+}
```

Comment:
```suggestion
make_check_pre="PYTHONPATH=\${PWD}/build/lib"
make_check_args="-k not(test_client_cert_password_verified)"
```
is should work just as well, although you might need `make_check_target="tests"`.

Reply:
Welp I fired from the hip here. I believe you can drop `make_check_pre` altogether and just use `make_check_args`.

Reply:
Done.